ui/shared/components/panel/acmg/AcmgCriteria.jsx

```diff
@@ -0,0 +1,221 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import { Table, Dropdown, Button, Message } from 'semantic-ui-react'
+import AcmgRuleSpecification from './AcmgRuleSpecification'
+import ACMG_DROP_DOWN_OPTIONS from './AcmgCriteriaDropDownOptions'
+import CATEGORY_CRITERIA_SCORE from './AcmgCategoryCriteriaScore'
+import { updateVariantClassification } from '../../../../redux/rootReducer'
+
+const FONT_STYLE_WRITING_MODE = { writingMode: 'sideways-lr', marginLeft: '-10px' }
+const TABLE_ROW_TOTAL_COLUMNS = 6
+const TABLE_COLUMN_COLOR = ['pink', 'blue', 'green', 'orange', 'red', 'red']
+
+export const getNewScoreValue = (criteria) => {
+  let newScore = 0
+  criteria.forEach((item) => {
+    newScore += CATEGORY_CRITERIA_SCORE[item]
+  })
+  return newScore
+}
+
+class AcmgCriteria extends React.PureComponent {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      formWarning: '',
+    }
+  }
+
+  setFormWarning(warning) {
+    this.setState({
+      formWarning: warning,
+    })
+  }
+
+  getCriteriaUsed() {
+    const criteriaUsed = {}
+    for (let i = 0; i < this.props.criteria.length; i++) {
+      criteriaUsed[this.props.criteria[i]] = true
+    }
```

Comment:
You do not need to change this, but as an FYI for future PRs our code style is to use ES 6 and map/ reduce instead of for loops. In the future it would be nice if you could conform to that style. So this would look something like 
```
return this.props.criteria.reduce((acc, criteria) => ({ ...acc, [criteria]: true }), {})
```